packages/console/src/utilities/icon.ts

```diff
@@ -0,0 +1,13 @@
+import { ApplicationType } from '@logto/schemas';
+
+import nativeAppIcon from '@/assets/images/native-app.svg';
+import singlePageAppIcon from '@/assets/images/single-page-app.svg';
+import traditionalWebAppIcon from '@/assets/images/traditional-web-app.svg';
+
+const appIconMapping = {
+  [ApplicationType.Native]: nativeAppIcon,
+  [ApplicationType.SPA]: singlePageAppIcon,
+  [ApplicationType.Traditional]: traditionalWebAppIcon,
+};
+
+export const getAppIcon = (appType: ApplicationType) => appIconMapping[appType];
```

Comment:
could just export `appIconMapping` ?

Reply:
I think `getAppIcon` is more meaningful.

Reply:
updated